cudart-sys/src/path.rs

```diff
@@ -0,0 +1,54 @@
+#[cfg(target_os = "windows")]
+#[macro_export]
+macro_rules! cuda_path {
+    () => {
+        env!("CUDA_PATH")
+    };
+}
+
+#[cfg(target_os = "linux")]
+#[macro_export]
+macro_rules! cuda_path {
+    () => {
+        "/usr/local/cuda"
```

Comment:
nit: to be more resilient against "custom" CUDA installs, we could extract the "path/to/cuda" piece of LD_LIBRARY_PATH, which the user must have set.

Reply:
> nit (suggestion): to be more resilient against "custom" CUDA installs, we could extract the `path/to/cuda` piece of `LD_LIBRARY_PATH`, which the user must have set.

That would not work with multiple CUDA toolkits installed because `LD_LIBRARY_PATH` would contain multiple paths to the respective toolkit installations but we need exactly one. The `/usr/local/cuda` symbolic link instead is supposed to point to the "default" version of the toolkit.
Also see https://cmake.org/cmake/help/latest/module/FindCUDAToolkit.html#search-behavior for how it is done by CMake. 
Trying to replicate CMake's behavior would be quite complicated, instead i implemented the base minimum that should work for majority of the cases.